src/layouts/layout.jsx

```diff
@@ -56,10 +58,42 @@ export const StyledFooterText = styled.label`
   size: 17px;
   line-height: 38px;    
 `
+const filters = [
+  {name: "all", display_name: "All"},
+  {name: "development", display_name: "Development"},
+  {name: "product-design", display_name: "Design"},
+  {name: "machine-learning", display_name: "Machine Learning"},
+  {name: "blockchain", display_name: "Blockchain"},
+  {name: "people-events", display_name: "People"},
+];
+
+const getPath = () => {
+  if (window.type !== "undefined") {
+    if (window.location.href === 'http://localhost:8000/') {
```

Comment:
what's the idea with this?

Reply:
The idea was to use that to make the default "all" filter, but, it only works on localhost and not the other URLs. I'm going to change that

Reply:
Done